server/public/styles.css

```diff
@@ -0,0 +1,259 @@
+* {
+  font-family: 'Glory', sans-serif;
+}
+
+body {
+  position: relative;
+}
```

Comment:
Since you are using bootstrap, you can probably remove most or all of your helper classes. 

https://getbootstrap.com/docs/5.1/utilities/background/

There is a large list of helper classes (starting with background color and going all the way through text alignment, flex, and position. Try to replace a large amount of your CSS with those.

There are also padding and margin helpers built into Bootstrap